tests/performance/local/native_tls_overhead.cpp

```diff
@@ -20,21 +20,17 @@
 #include <thread>
 #include <vector>
 
-#if defined(__has_feature)
```

Comment:
bad lose `if`
also when `__has_feature` is not defined then `__has_feature(cxx_thread_local)` is ill-formed as it evaluates to `0(0)`

Reply:
Good catch. This whole block should go. I now assume `thread_local` is a keyword, and if not compilation will fail.

Reply:
Deleted this whole block.